curvefs/src/client/fuse_client.h

```diff
@@ -217,6 +226,50 @@ class FuseClient {
     SpaceBaseClient *spaceBase_;
 };
 
+class RenameOperator {
```

Comment:
put this class in another file will be better.

Reply:
> put this class in another file will be better.

fixed.